chart/README.md

```diff
@@ -0,0 +1,49 @@
+## OpenFaaS Cloud
+
+This is the experimental Helm Chart for installing OpenFaaS Cloud.
+
+It is currently not recommended to use this chart directly, it is intended to be used as part of `ofc-bootstrap`
+to install OpenFaaS Cloud clusters.
+
+This is mainly for configuration documentation for developers wising to use this chart in `ofc-bootstrap`.
```

Comment:
"wising" --> "wishing"

Reply:
Thanks :+1: 
My spellchecker didnt pick that up for some reason :thinking: